### DISCUSSION TITLE
Can we give QOS to OTT Voice Traffic over RAN?

### DISCUSSION THREAD

**Discussion by shehzadalee** [magma/magma#6879] [Q&A]
We are trying to give QoS to OTT voice traffic over the RAN. Example traffic is SIP port 5060 to destination IP 192.168.100.100, with DSCP = 46. Is there any guide on how we would do this for magma network?
These is network type “LTE” without an external OCS.

**Comment by karthiksubraveti** [comment]
We can associate APN level QOS which can set QOS to OTT voice traffic over the RAN. AFAIK, the flow level QOS is a local property and is only applicable within the gateway. cc @ulaskozat
